index.html

```diff
@@ -1,188 +1,202 @@
+                        <li>css</li>
+                        <li>javaScript</li>
                     </ul>
-                </section>
-                <p>
-                    A daily selection of privately personalized 
-                    reads; no accounts or sign-ups required.
-                </p>
-                <ul class="programmingLanguages">
-                    <li>html</li>
-                    <li>css</li>
-                    <li>javaScript</li>
-                </ul>
-                <button class="button">See Project</button>
+                    <button class="button">See Project</button>
             </section>
-            <!-- projecto 4 -->
+            <!-- project 4 -->
             <section class="project">
-                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 4.svg" alt="Project picture">
+                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 4.svg" alt="Fourth project card picture">
                 <h2>Multi-Post Stories</h1>
-                <section>
-                    <ul class="canopy">
-                        <li>CANOPY</li>
-                        <li><img src="img\Counter.svg"></li>
-                        <li><span class="lightgray">Back End Dev</span></li>
-                        <li><img src="img\Counter.svg"></li>
-                        <li><span class="lightgray">2015</span></li>
+                    <section>
+                        <ul class="canopy">
+                            <li>CANOPY</li>
+                            <li><img src="img\Counter.svg" alt="Dot icon"></li>
+                            <li><span class="lightgray">Back End Dev</span></li>
+                            <li><img src="img\Counter.svg" alt="Dot icon"></li>
+                            <li><span class="lightgray">2015</span></li>
+                        </ul>
+                    </section>
+                    <p>
+                        A daily selection of privately personalized
+                        reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="programmingLanguages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
                     </ul>
-                </section>
-                <p>
-                    A daily selection of privately personalized 
-                    reads; no accounts or sign-ups required.
-                </p>
-                <ul class="programmingLanguages">
-                    <li>html</li>
-                    <li>css</li>
-                    <li>javaScript</li>
-                </ul>
-                <button class="button">See Project</button>
+                    <button class="button">See Project</button>
             </section>
         </section>
         <section class="presentation" id="aboutContainer">
             <section class="content" id="about">
                 <h1>About<br>Myself</h1>
```

Comment:
- [ ] The page has only one heading `<h1>` please kindly change your `h1` to another `hx` tag. `eg: <h2>About<br>Myself</h2>` 👍 👍